From c213ffa2e0ba6677e0dfade9c67b91446c70c8bd Mon Sep 17 00:00:00 2001
From: Jun Zhang <jun.zhang@windriver.com>
Date: Fri, 16 Oct 2015 21:25:17 +0800
Subject: [PATCH] cpio: fix testcase symlink-bad-lengths

* fix the output of stderr & stdout to meet latest cpio

Upstream-Status: Pending

Signed-off-by: Jun Zhang <jun.zhang@windriver.com>
---
 tests/symlink-bad-length.at | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tests/symlink-bad-length.at b/tests/symlink-bad-length.at
index 6f804b1..74208ce 100644
--- a/tests/symlink-bad-length.at
+++ b/tests/symlink-bad-length.at
@@ -41,9 +41,8 @@ cpio -ntv < ARCHIVE
 test $? -eq 2
 ],
 [0],
-[-rw-rw-r--   1 10029    10031          13 Nov 25 13:52 FILE
-],[cpio: LINK: stored filename length too big
-cpio: premature end of file
+[-rw-rw-r--   1 10029    10031          13 Nov 25  2014 FILE
+],[cpio: premature end of file
 ])
 
 AT_CLEANUP
-- 
1.9.1

